ConEmuWinForms/ConEmuSession.cs

```diff
@@ -501,6 +495,12 @@ private static unsafe CommandLineBuilder Init_MakeConEmuCommandLine([NotNull] Co
 			}
 			cmdl.AppendSwitchIfNotNull("-cur_console:", $"{(startinfo.IsElevated ? "a" : "")}{sConsoleExitMode}");
 
+			// ANSI Log file
+			if (ansilog != null)
+			{
+				cmdl.AppendSwitchIfNotNull("-new_console:L:", ansilog.Directory.FullName);

```

Comment:
new_console is working, but I believe that cur_console should be used
```suggestion
				cmdl.AppendSwitchIfNotNull("-cur_console:L:", ansilog.Directory.FullName);

```

Reply:
I just tried, both works. I have no idea if it's new_console or cur_console.